spec/features/new_project_spec.rb

```diff
@@ -223,6 +223,43 @@
     expect(File).to exist("#{project_path}/spec/factories.rb")
   end
 
+  it "creates the local heroku setup" do
+    bin_setup_path = "#{project_path}/bin/setup_review_app"
+    bin_setup = IO.read(bin_setup_path)
+
+    expect(bin_setup).to include("heroku run rake db:migrate --exit-code --app #{app_name.dasherize}-staging-pr-$1")
+    expect(bin_setup).to include("heroku ps:scale worker=1 --app #{app_name.dasherize}-staging-pr-$1")
+    expect(bin_setup).to include("heroku restart --app #{app_name.dasherize}-staging-pr-$1")
+    expect(File.stat(bin_setup_path)).to be_executable
+
+    bin_deploy_path = "#{project_path}/bin/deploy"
+    bin_deploy = IO.read(bin_deploy_path)
+
+    expect(bin_deploy).to include("heroku run rake db:migrate --exit-code")
+    expect(File.stat(bin_deploy_path)).to be_executable
+
+    circle_yml_path = "#{project_path}/circle.yml"
+    circle_yml = IO.read(circle_yml_path)
+
+    expect(circle_yml).to include <<-YML.strip_heredoc
+      deployment:
+        staging:
+          branch: master
+          commands:
+            - bin/deploy staging
+    YML
+  end
+
+  it "adds app.json file" do
+    expect(File).to exist("#{project_path}/app.json")
+  end
```

Comment:
> Also, I think that the app.json test should be inside the "creates the local heroku setup" block, as it is part of it. What do you think?

That's true, but I like having smaller specs with more specific descriptions. I'd leave them separate (even though the project does not follow this rule in the different spec files).